<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExtractDialog</class>
 <widget class="QDialog" name="ExtractDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Extract</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="extractToLabel">
       <property name="text">
        <string>Extract to:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="pathComboBox">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="applyPathPushButton">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>-&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="singleFolderGroupBox">
         <property name="title">
          <string>Extract into subfolder</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLineEdit" name="subfolderLineEdit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="filesToExtractGroupBox">
         <property name="title">
          <string>Files</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="allFilesRadioButton">
            <property name="text">
             <string>All files</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="selectedFilesRadioButton">
            <property name="text">
             <string>Selected files only</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="optionsGroupBox">
         <property name="title">
          <string>Options</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="preservePathsCheckBox">
            <property name="toolTip">
             <string>Extract pathnames</string>
            </property>
            <property name="whatsThis">
             <string>If checked QArchiver extract pathnames</string>
            </property>
            <property name="text">
             <string>Extract paths</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="openFolderCheckBox">
            <property name="text">
             <string>Open folder after extracting</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="closeAfterCheckBox">
            <property name="text">
             <string>Close after extraction</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="overwriteExistingCheckBox">
            <property name="text">
             <string>Overwrite existing files</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="fsExplorerLabel">
         <property name="text">
          <string>Files/Folders:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="treeView"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="newDirPushButton">
           <property name="text">
            <string>New dir...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="extractPushButton">
       <property name="text">
        <string>Extract</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/actions/archive-extract</normaloff>:/actions/archive-extract</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/actions/dialog-cancel</normaloff>:/actions/dialog-cancel</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pathComboBox</tabstop>
  <tabstop>applyPathPushButton</tabstop>
  <tabstop>treeView</tabstop>
  <tabstop>subfolderLineEdit</tabstop>
  <tabstop>allFilesRadioButton</tabstop>
  <tabstop>selectedFilesRadioButton</tabstop>
  <tabstop>preservePathsCheckBox</tabstop>
  <tabstop>openFolderCheckBox</tabstop>
  <tabstop>overwriteExistingCheckBox</tabstop>
  <tabstop>newDirPushButton</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>extractPushButton</sender>
   <signal>clicked()</signal>
   <receiver>ExtractDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>56</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>ExtractDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
